crates/cargo-test-support/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cargo-test-support"
-version = "0.7.1"
+version = "0.7.2"
```

Comment:
Should you also update https://github.com/rust-lang/cargo/blob/684bca20121f66a4fc7b02638222f50cf9cd8250/Cargo.toml#L36